Dockerfile

```diff
@@ -3,8 +3,8 @@ RUN apk add --no-cache curl jq
 RUN mkdir -p /assets
 WORKDIR /assets
 RUN curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github-rel" | tar -xzf -
-RUN url=$(curl -s "https://api.github.com/repos/contraband/autopilot/releases/latest" \
```

Comment:
Are you going to merge your changes (adding tail logs) to upstream of `autopilot`? Would that be enough to get things going here in this PR?

Reply:
We've sent the changes [upstream](https://github.com/contraband/autopilot/pull/43) and also submitted a number of [other fixes](https://github.com/contraband/autopilot/pull/44) to autopilot, however unfortunately the project seems to be starved of attention by the maintainers.

FWIW we've been using our forked version for all new projects for the past 4 weeks or so and it's been working well for us.